packages/patternfly-4/react-core/src/layouts/Page/__snapshots__/PageHeader.test.js.snap

```diff
@@ -19,6 +19,7 @@ exports[`Check page vertical layout example against snapshot 1`] = `
 .pf-l-page__header.pf-m-tall {
   display: flex;
   grid-area: header;
+  z-index: 300;
```

Comment:
I'm really interested in why our snapshots need to reference direct css values especially since the react repo doesn't control these.

Reply:
I think it's useful to see, when we upgrade Core, what visually has changed for the components

Reply:
The main idea for this before was to catch things core may have missed since there is no unit testing in core. It has caught issues with variables being referenced that didn't actually exist before. This is why variables are replaced with raw values. This was all put in as a stop gap or alternative to visual regression tests.